manager/README.md

```diff
@@ -55,6 +55,22 @@ VAULT_MODULES_ROLE="module"
 ENABLE_WEBHOOKS="false"
 ```
 
+If the manager runs in a Razee environment, you also need to add the following environment variables:
```

Comment:
Hi thanks for the PR, 
but it is not clear what do you mean by "a Razee environment", I assume that you mean Razee and not Satellite deployment. However, the Raze can be collocated on the same cluster as the manager, or can be a remote service. Manager can connect to Razee by using userName/password  as you explain here or by using oauth API key, as shown [here](https://fybrik.io/v0.6/tasks/multicluster/#coordinator-cluster).

When a user (developer)  runs  manager manually, he probably knows knows which Razee he plans to use. 
I would suggest the following changes:
If the manager will works with a Razee service,  you also need to add the following environment variables:
```bash
RAZEE_URL=<Razee access point> # e.g. "http://localhost:3333/graphql"
RAZEE_USER=<Razee user>
# you should define either RAZEE_PASSWORD or API_KEY, if both variables are defined RAZEE_PASSWORD will be used
RAZEE_PASSWORD=<Razee password> 
API_KEY=<Razee api key>
```
And the paragraph `You can know that the manager runs in a Razee environment by` (Lines 66- 73) can be removed.

Reply:
Thanks! I will update it with your suggestions.